tests/rhtap-demo/rhtap-demo.go

```diff
@@ -118,6 +113,10 @@ var _ = framework.RhtapDemoSuiteDescribe(Label("rhtap-demo"), Label("verify-stag
 
 			Describe(appTest.Name, Ordered, func() {
 				BeforeAll(func() {
+					if token == "" && ssourl == "" && apiurl == "" && username == "" {
```

Comment:
oh, sorry I meant it like this (because we want to validate env var values' presence only in case we run "verify-stage" test)
```suggestion
if Label("verify-stage").MatchesLabelFilter(GinkgoLabelFilter()) {
					if token == "" && ssourl == "" && apiurl == "" && username == "" {
```

Reply:
Thanks! I did something, PTAL.

btw a question about [this line](https://github.com/redhat-appstudio/e2e-tests/blob/1344aa2aa76e78c3511c6d0fba029712da01bb74/tests/rhtap-demo/rhtap-demo.go#L80): what does this `Label("rhtap-demo"), Label("verify-stage")` do exactly? Does it add both these labels to the test suite? If yes, then what's the point of:
* `if Label("verify-stage").MatchesLabelFilter(GinkgoLabelFilter())` and
* `if Label("rhtap-demo").MatchesLabelFilter(GinkgoLabelFilter())`
wouldn't these both be true?

Reply:
It is not ideal. I'm trying to find a better to approach to this, which will be addressed in https://issues.redhat.com/browse/RHTAPBUGS-1118